deepinv/transform/shift.py

```diff
@@ -8,11 +8,13 @@ class Shift(torch.nn.Module):
     Generates n_transf randomly shifted versions of 2D images with circular padding.
 
     :param n_trans: number of shifted versions generated per input image.
+    :param float shift_max: maximum shift as fraction of total height/width.
     """
 
-    def __init__(self, n_trans=1):
+    def __init__(self, n_trans=1, shift_max=1):
```

Comment:
a pedantic comment, but it would be good to set it as 1. to stress the float type

Reply:
Done